1351.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {number[][]} grid
+ * @return {number}
+ */
+var countNegatives = function (grid) {
+  let negativeSum = 0;
+
+  for (let i = 0; i < grid.length; i++) {
+    for (let j = 0; j < grid[i].length; j++) {
+      if (grid[i][j] < 0) {
+        negativeSum++;
+      }
+    }
+  }
```

Comment:
You can use one for-loop by using `Array.flat()` instead of nested for-loops
You do not need to change your code but it's good to know the method 👍